openapi.yaml

```diff
@@ -0,0 +1,191 @@
+openapi: '3.0.0'
+
+info:
+  title: GOV.UK Content API
+  description:
+  version: '1.0.0'
+
+servers:
+  - url: https://www.gov.uk/api/content
+
+paths:
+  /{path}:
+    get:
+      summary: Access a content item by path
+      description: The primary means to access content that is hosted on GOV.UK.
+      responses:
+        200:
+          description: A content item is available at that path.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ContentItem'
+        404:
+          description: No content item is available at that path.
+
+components:
+  schemas:
+
+    WithdrawnNotice:
+      required:
+        - explanation
+        - withdrawn_at
+      properties:
+        explanation:
+          type: string
+          description: An explanation as to why the content was withdrawn.
+        withdrawn_at:
+          type: string
+          format: date-time
+          description: The date when the content was withdrawn.
+
+    LinkedContentItem:
+      required:
+        - analytics_identifier
+        - api_path
+        - base_path
+        - content_id
+        - description
+        - document_type
+        - locale
+        - public_updated_at
+        - schema_name
+        - title
+        - withdrawn
```

Comment:
I believe not all of these are required: https://github.com/alphagov/govuk-content-schemas/blob/master/formats/base_path_less_frontend_links_definition.json